janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/JanusGraphIndexTest.java

```diff
@@ -1115,21 +1115,29 @@ public void testGraphCentricQueryProfiling() {
         final PropertyKey prop = makeKey("prop", String.class);
         final PropertyKey description = makeKey("desc", String.class);
         final PropertyKey pet = makeKey("pet", String.class);
-        mgmt.buildIndex("mixed", Vertex.class).addKey(name, Mapping.STRING.asParameter())
-            .addKey(prop, Mapping.STRING.asParameter()).buildMixedIndex(INDEX);
+        final PropertyKey height = makeKey("height", Integer.class);
+        mgmt.buildIndex("mixed", Vertex.class)
+            .addKey(name, Mapping.STRING.asParameter())
+            .addKey(prop, Mapping.STRING.asParameter())
+            .addKey(height)
+            .buildMixedIndex(INDEX);
         mgmt.buildIndex("mi", Vertex.class).addKey(description).addKey(pet).buildMixedIndex(INDEX2);
         finishSchema();
 
-        tx.addVertex("name", "bob", "prop", "val", "desc", "he likes coding", "pet", "he likes dogs", "age", 20);
-        tx.addVertex("name", "bob", "prop", "val2", "desc", "he likes coding", "pet", "he likes cats", "age", 25);
-        tx.addVertex("name", "alex", "prop", "val", "desc", "he likes debugging", "pet", "he likes cats", "age", 20);
+        tx.addVertex("name", "bob", "prop", "val", "desc", "he likes coding", "pet", "he likes dogs", "age", 20, "height", 170);
+        tx.addVertex("name", "bob", "prop", "val2", "desc", "he likes coding", "pet", "he likes cats", "age", 25, "height", 175);
+        tx.addVertex("name", "alex", "prop", "val", "desc", "he likes debugging", "pet", "he likes cats", "age", 20, "height", 190);
         tx.commit();
 
         // satisfied by a single graph-centric query which is satisfied by a single mixed index query
         if (indexFeatures.supportNotQueryNormalForm()) {
             newTx();
             assertEquals(3, tx.traversal().V().or(__.has("name", "bob"), __.has("prop", "val")).count().next());
             assertEquals(3, tx.traversal().V().or(__.has("name", "bob"), __.has("prop", "val")).toList().size());
+            assertEquals(190, tx.traversal().V().or(__.has("name", "bob"), __.has("prop", "val")).values("height").max().next());
```

Comment:
Could you please add one more test case to cover the branch mentioned [here](https://github.com/JanusGraph/janusgraph/pull/3203#discussion_r985127606)? I know the branch has been removed, but right now that default case is still not tested. An example would be:
```java
assertEquals(25, tx.traversal().V().or(__.has("name", "bob"), __.has("prop", "val")).values("age").max().next());
```
i.e. A mixed index is matched, but the index itself does not cover the property of interest. In other words, the mixed index is not a covering index.

Reply:
I've added a [test](https://github.com/StrangeBeeCorp/janusgraph/blob/aggregations/janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/IndexMetricTest.java#L71) that cover the default case, but only for metrics (the result is not checked).
I'll add this test.